Tasks/VSBuild/make.json

```diff
@@ -0,0 +1,7 @@
+{
+    "common": [
+        {
+            "module": "../Common/MSBuildHelpers"
```

Comment:
i thought about adding `type=ps`, but then thought i may as well call it `type=copyProj` instead. since it's not used for anything for ps modules, i ended up not adding `type` at all.